CHANGELOG.md

```diff
@@ -4,6 +4,24 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 0.9.1 (2018-03-26)
+
+**High-level changes**
+* Several edge cases regarding CTEs addressed
+* Updated preloader with better error messaging and fixed edge case
+
+**Notable commits**
+* [40ce037] Fix crash on explain analyze with insert cte
+* [8378beb] Enable hypertable inserts within CTEs
+* [bdfda75] Fix double-loading of extension
+* [01ea77e] Fix EXPLAIN output for ConstraintAwareAppend inside CTE
+* [fc05637] Add no preload error to versioned library.
+* [38f8e0c] Add ABI compatibility tests
+* [744ca09] Fix Cache Pinning for Subtxns
+* [39010db] Move more drops into event trigger system
+* [fc36699] Do not fail add_dimension() on non-empty table with 'if_not_exists'
+
```

Comment:
I think there should be some **thanks**?

Reply:
Oops yeah